truss/remote/baseten/remote.py

```diff
@@ -239,69 +237,80 @@ def patch(
         self,
         watch_path: Path,
     ):
+        from truss.cli.console import console
+
         try:
             truss_handle = TrussHandle(watch_path)
         except yaml.parser.ParserError:
-            logger.error("Unable to parse config file")
+            console.print("Unable to parse config file", style="red")
             return
         except ValueError:
-            logger.error(
-                f"Error when reading truss from directory {watch_path}", exc_info=True
+            console.print(
```

Comment:
we should consider using an error console with a default style for this kind of exception logging
https://rich.readthedocs.io/en/stable/console.html#error-console